### PR TITLE
Release v1.5.0-rc2

### PR DESCRIPTION
**Pull request by marcelstanley**

### CODE REVIEW

**Review by vfusco — DISMISSED** [review]

File: docs/upgrade_to_1_5.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Where `BLOCK_TIME` corresponds to the duration it takes to generate a block in t
 >
 > So, `CARTESI_EPOCH_LENGTH = 86400/12 = 7200`
 
-Check the [`CHANGELOG`](./CHANGELOG.md) and the [configuration](./docs/config.md) for more details.
+Check the [`CHANGELOG`](../CHANGELOG.md) and the [Rolups Node configuration](./config.md) for more details.
```
Comment (GMKrieger) [inline docs/upgrade_to_1_5.md:29]: RolLups with 2 L's

File: docs/upgrade_to_1_5.md [hunk 2]
```diff
@@ -105,6 +105,10 @@ cast send \
     --rpc-url "$RPC_URL"
 ```
 
+> [!NOTE]
+> `cast send` will fail if the `History` type is not known to `cast` at the time the of the execution.
```
Comment (GMKrieger) [inline docs/upgrade_to_1_5.md:109]: At the end there's one too many `the`

`at the time of the execution.`

**Review by renan061 — DISMISSED** [review]

File: docs/upgrade_to_1_5.md (hunk 1 above)
Comment (marcelstanley) [reply docs/upgrade_to_1_5.md]: Done at 10cd9ce9e0878237bfa9f7b940d4ec30f3c50052

File: docs/upgrade_to_1_5.md (hunk 2 above)
Comment (marcelstanley) [reply docs/upgrade_to_1_5.md]: Done at 10cd9ce9e0878237bfa9f7b940d4ec30f3c50052

**Review by renan061 — APPROVED** [review]

**Review by GMKrieger — APPROVED** [review]

**Pull request merged**